content/docs/working-groups/learning/minutes/_index.md

```diff
@@ -0,0 +1,5 @@
+# Learning Minutes
+
+- [Agenda]({{< relref "/docs/working-groups/learning/minutes/agenda" >}})
+- [2019.05.14 Meeting]({{< relref "/docs/working-groups/learning/minutes/2019.05.14-meeting" >}})
```

Comment:
Similarly with the triage and steering meeting pages, I think it would be best to avoid manually-maintained index pages like this.

Reply:
David, I have a question on this issue. Would you like us to add the links to the sidebar menu? If we add all the meetings in the sidebar it would be a very long list, is that ok for you? We could also add the links to the index page of the working group, but that would be a list of links again

Reply:
I don’t mind either approach. I think you’re right that if they were all on the sidebar then that could get quite long, so I think having them on the index page for working groups would also be good. I was mostly hoping to avoid having a page that was only a list of links that were just dates.